runtime/metricsview/query.go

```diff
@@ -94,6 +98,11 @@ type MeasureComputeComparisonRatio struct {
 	Measure string `mapstructure:"measure"`
 }
 
+type MeasureComputePercentOfTotal struct {
+	Measure string  `mapstructure:"measure"`
+	Total   float64 `mapstructure:"total"`
```

Comment:
Change to `*float64` to distinguish between cases where the total was not computed, and where the total is `0`